server/graphql/common/expenses.ts

```diff
@@ -1475,7 +1476,23 @@ async function payExpenseWithPayPalAdaptive(remoteUser, expense, host, paymentMe
     }
 
     // Warning senderFees can be null
-    const senderFees = createPaymentResponse.defaultFundingPlan.senderFees;
+    let senderFees = createPaymentResponse.defaultFundingPlan.senderFees;
+    if (!senderFees) {
+      // PayPal stopped providing senderFees in the response, we need to compute it ourselves
+      // We don't have to check for feesPayer here because it is not supported for PayPal adaptive
+      const { fundingAmount } = createPaymentResponse.defaultFundingPlan;
+      const amountPaidByTheHost = floatAmountToCents(parseFloat(fundingAmount.amount));
+      const amountReceivedByPayee = expense.amount;
```

Comment:
No multi-currency issues here?

Reply:
Because adaptive is now disabled for multi-currency cases?

Reply:
Exactly, we can take this shortcut since multi-currency is disabled for adaptive. The check below (`if (fundingAmount.code !== expense.currency)`) with a Sentry notification is a way to be notified if, somehow, this rule ever gets broken.